proposals/csharp-9.0/target-typed-new.md

```diff
@@ -2,7 +2,7 @@
 # Target-typed `new` expressions
```

Comment:
https://github.com/dotnet/roslyn/blob/master/docs/Language%20Feature%20Status.md

This states "Merged into 16.7p1"